### DISCUSSION TITLE
Empty index page and still showing deleted default articles in archives

### DISCUSSION THREAD

**Discussion by josyulavt** [cotes2020/jekyll-theme-chirpy#1313] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

Hey all, 
1) I have just started using this theme and experimenting with it, I have added some content to my about.md page 

```
---
# the default layout is 'page'
icon: fas fa-info-circle
order: 4
---

Hi! Welcome to my personal blog


```

when I compile this  locally using `bundle exec jekyll s --incremental` it does work as expected but when I navigate to https://josyulavt.github.io/ it just shows `"--- layout: home # Index page ---"` however rest of the pages seem to be working ok 
https://josyulavt.github.io/archives/ when I click home this happens. 

2) I have deleted some default posts that come with the theme and I still see them appearing in the archives (locally).

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/139254060/acd67bc5-51c8-4600-9d36-a32f6db5701b)

Any help is very much appreciated

### Operations you have already tried

1) I have changed the code in pages-deploy.html to 
```

on:
  push:
    branches:
      - master
    paths-ignore:
      - .gitignore
      - README.md
      - LICENSE
```

2) I have also tried changing the layout to `layout:page ` which also doesnt work 

please help. thank you 

3) added url in _config.yml `url: "https://josyulavt.github.io"`

**Comment by DmitriyFrogo** [comment]
Oh, I see, that you just forked this repo
Seems like, you haven't read the manual =)

Read this if you like difficulties
https://chirpy.cotes.page/posts/getting-started/#option-2-github-fork

If not, I recommend you to backup your old repo and generate new one from chirpy-starter
https://chirpy.cotes.page/posts/getting-started/#option-1-using-the-chirpy-starter

**Reply by josyulavt** [reply]
Thank you Dmitiry I have already set it GitHub Actions.. 


Let me see i can do the chirpy starter thing..
